library/rsa.c

```diff
@@ -1228,6 +1228,16 @@ int mbedtls_rsa_check_pub_priv(const mbedtls_rsa_context *pub,
     return 0;
 }
 
+/*
+ * This function is identical to mbedtls_mpi_exp_mod() the only difference is that this function is
+ * not constant time.
+ *
+ * WARNING! This function is not constant time.
+ */
+int mbedtls_mpi_exp_mod_unsafe(mbedtls_mpi *X, const mbedtls_mpi *A,
```

Comment:
Shouldn't this have `extern` on it?

Reply:
I thought that all function declarations are implicitly `extern`

Reply:
Hmm, yes, and we seem to use that in Mbed TLS, so I'll withdraw this comment (although it looks strange to me without `extern` since we're in a `.c` file)